_posts/2015-03-18-Wifi-pairing.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: Wifi pairing
+---
+Earlier I suggested that the Known groups for Wifi Direct could be teached for the device.
+Thus making the Wifi direct "pairable" same way as Bluetooth.
```

Comment:
I found this sentence hard to follow. I think what you mean is something like "Wifi Direct on Android requires user confirmation to establish a connection. But as explained [here](http://www.drjukka.com/blog/wordpress/?p=46) we can treat this like Bluetooth pairing. Once two devices are 'paired' they can communicate over Wifi Direct in the background or the foreground without any further user confirmations! This opens up Wifi Direct's longer range and higher bandwidth as an alternative to unauthenticated Bluetooth connections."